src/communication/network_interface_sr.c

```diff
@@ -7268,6 +7269,208 @@ slogtb_dump_trantable (THREAD_ENTRY * thread_p, unsigned int rid, char *request,
   db_private_free_and_init (thread_p, buffer);
 }
 
+/*
+ * smmon_get_module_info -
+ *
+ * return:
+ *
+ *   rid(in):
+ *   request(in):
+ *   reqlen(in):
+ */
+void
+smmon_get_module_info (THREAD_ENTRY * thread_p, unsigned int rid, char *request, int reqlen)
+{
+  char *buffer, *ptr;
+  int size = 0;
+  OR_ALIGNED_BUF (OR_INT_SIZE + OR_INT_SIZE) a_reply;
+  char *reply = OR_ALIGNED_BUF_START (a_reply);
+  int error = NO_ERROR;
+  MMON_MODULE_INFO info;
+  MMON_COMP_INFO *c_info;
+  MMON_SUBCOMP_INFO *s_info;
```

Comment:
```suggestion
  MMON_MODULE_INFO module_info;
  MMON_COMP_INFO *comp_info;
  MMON_SUBCOMP_INFO *subcomp_info;
```
how about using a meaningful names?